app/serializers/invite_serializer.rb

```diff
@@ -1,21 +1,39 @@
 # frozen_string_literal: true
 
 class InviteSerializer < ApplicationSerializer
```

Comment:
It seems like we need to rename this serializer since it isn't a serializer for the `Invite` model anymore and is a serializer for `InvitedUser`

Reply:
That is not entirely correct. It serializes `Invite` model in case of "pending" tab. I'd prefer to leave it as is.

Reply:
Ooo this is confusing because a serializer can serialize two different models. I think we should split the serializer into two and call the right serializer in the controller rather than have one serializer handle this.

Reply:
Okay, I split the serializer in two. Which serializer to use will be decided in `InvitedSerializer` based on object type.